### ISSUE TITLE
new logo?

### ISSUE THREAD

**Issue by leandroebner** [GameTec-live/ChameleonUltraGUI#489]
hey. This is super low level priority but what is the app icon actually displaying?
I thought its a chameleon but I cant really see it. It always just looks like a broken image file. 
That would also improve the overall appearance in the app-store (play-store) and maybe attract more people. 
Just a thought for the far future, Id like to help out as well if a design is needed!

Cheers, thanks for that great project!

**Comment by GameTec-live** [comment]
Yeah, its supposed to be the Chameleon in some weird rainbiw particle thing. I know, its horrible, but im no graphics designer, so its just something i chucked together in blender when i was bored.
If you know how to do graphics, feel free to try and make a logo, if its great, maybe itll become the new logo ;)

**Comment by leandroebner** [comment]
no worries!
perfect, Ill keep that in mind when Im doing other design work too :)

**Comment by nasaboy** [comment]
Looking forward to your design of a new logo.

**Comment by Saint-Ricky** [comment]
Not sure if you can use one or both of these but I thought I’d give it a go. 
![IMG_0571](https://github.com/user-attachments/assets/c5a6c3fb-ed1e-409b-99f6-a83931d2090b)
![IMG_0570](https://github.com/user-attachments/assets/a6ef6756-5c54-499a-ba13-05cb0461854a)

**Comment by GameTec-live** [comment]
@Saint-Ricky Dont want to be mean, but they do look to close to the gold printing on the front side of the CU for my taste. The app should still have its own identity while drawing a connection to the CU,.

**Comment by Saint-Ricky** [comment]
@GameTec-live Yeah I wasn’t sure if that would be an issue or not. That was what I was going for at the time. I also threw this one together which is more abstractly a chameleon but with some radio signal coming off. Let me know what you think. I’m definitely open to suggestions and input. 
![IMG_0577](https://github.com/user-attachments/assets/037919d6-62ee-4b77-b837-8a2bca6cd44d)

**Comment by GameTec-live** [comment]
Thanks to the extremely talented @Saint-Ricky theres work on a new CUGUI logo. Please tell me which direction you like best or, if nothing fits, whats wrong with the existing ones / what youd like to see.

![4](https://github.com/user-attachments/assets/ae6c744f-657b-4d12-926a-418c833bc683)
![3](https://github.com/user-attachments/assets/65bec0b6-637b-44f0-91fd-29af1b727115)
![2](https://github.com/user-attachments/assets/7dad3d82-a94a-442b-8602-a5c2d4149919)
![1](https://github.com/user-attachments/assets/e5845363-477b-43eb-9bc2-c315aff7ecc4)
![5](https://github.com/user-attachments/assets/660b4d6e-3726-43a4-bede-092de96b0767)

**Comment by duchenpaul** [comment]
> @GameTec-live Yeah I wasn’t sure if that would be an issue or not. That was what I was going for at the time. I also threw this one together which is more abstractly a chameleon but with some radio signal coming off. Let me know what you think. I’m definitely open to suggestions and input. 
> 
> ![IMG_0577](https://github.com/user-attachments/assets/037919d6-62ee-4b77-b837-8a2bca6cd44d)
> 
> 

Actually I like this design! Though it looks like AI artwork, but love it at the first glance
